src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -20,273 +20,692 @@ PICS:
+                value: 0x0002
+              - name: "GroupName"
+                value: "Group2"
+      response:
+          values:
+              - name: "Status"
+                value: 0
+              - name: "GroupID"
+                value: 0x0002
 
     - label:
           "TH sends a RemoveAllScenes command to DUT with the GroupID field set
           to 0x0002."
       PICS: S.S.C03.Rsp
-      verification: |
-          DUT sends a RemoveAllScenesResponse command to TH with the Status field set to 0x00 (SUCCESS) and GroupID field set to 0x0002.
-      disabled: true
+      command: "RemoveAllScenes"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0002
+      response:
+          values:
+              - name: "Status"
+                value: 0
 
     - label:
           "TH sends a GetSceneMembership command to DUT with the GroupID field
           set to 0x0002."
       PICS: S.S.C06.Rsp
-      verification: |
-          DUT sends a GetSceneMembershipResponse command to TH with the Status field set to 0x00 (SUCCESS), the Capacity field set to the value SC1, the GroupID field set to 0x0002 and the SceneCount field set to 0x00.
-      disabled: true
+      command: "GetSceneMembership"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0002
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
+              - name: "Capacity"
+                value: 6
+              - name: "GroupID"
+                value: 0x0002
 
     - label:
           "TH configures AC3 on DUT for all implemented application clusters
           supporting scenes."
-      verification: |
-          DUT is configured with AC3 which is different to both AC1 and AC2.
-      disabled: true
+      cluster: "Level Control"
+      command: "MoveToLevelWithOnOff"
+      arguments:
+          values:
+              - name: "Level"
+                value: 1
+              - name: "TransitionTime"
+                value: 0
+              - name: "OptionsMask"
+                value: 0
+              - name: "OptionsOverride"
+                value: 0
 
     - label:
           "TH sends a StoreScene command to DUT with the GroupID field set to
           0x0002 and the SceneID field set to 0x03."
       PICS: S.S.C04.Rsp
-      verification: |
-          DUT sends a StoreSceneResponse command to TH with the Status field set to 0x00 (SUCCESS) or 0x89 (INSUFFICIENT_SPACE). If SUCCESS, with the GroupID field set to 0x0002 and the SceneID field set to 0x03. IF INSUFFICIENT_SPACE, the following steps SHALL NOT be executed.
-      disabled: true
+      command: "StoreScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: 0x0002
+              - name: "SceneID"
+                value: 0x03
+      response:
+          values:
+              - name: "Status"
+                value: 0x00
```

Comment:
Test plan allows INSUFFICIENT_SPACE.

Reply:
Same as above, I allowed it in the test but how should the test handle the leave on insufficient space?